client/src/components/Donation/DonateForm.js

```diff
@@ -157,7 +157,7 @@ class DonateForm extends Component {
   }
 
   convertToTimeContributed(amount) {
-    return `${numToCommas((amount / 100) * 50)} hours`;
+    return `${numToCommas((amount / 100) * 50)}`;
```

Comment:
There is no need to use Template Literals after changes:

`return numToCommas((amount / 100) * 50);`

would be better

Reply:
Remove Template Literals.